src/app/public/modules/ui-config/ui-config.service.ts

```diff
@@ -7,7 +7,9 @@ import {
   of as observableOf
 } from 'rxjs';
 
-@Injectable()
+@Injectable({
+  providedIn: 'root'
+})
 export class SkyUIConfigService {
```

Comment:
Same question as above - Why wasn't `SkyUIConfigService` being included in any modules before?

Reply:
The guidance for `SkyUIConfigService` (as it stands today) is to always overwrite the provider in your application, see:
https://developer.blackbaud.com/skyux/learn/get-started/advanced/sticky-settings/integrations/custom-components. However, we want this service to be tree-shakeable in case it is imported as-is.